cloudferrylib/os/actions/check_users.py

```diff
@@ -28,6 +28,9 @@ class CheckUsersAvailabilityOnSrcAndDst(action.Action):
     """
 
     def run(self, **kwargs):
+        if self.cfg.migrate.migrate_users:
+            LOG.info("Users will be migrated. Skipping this check.")
+            return
```

Comment:
This seem unrelated to the patch, might be better to submit as a separate pull request